common/management/commands/create_test_data.py

```diff
@@ -0,0 +1,1775 @@
+            # future
+            (
+                datetime.now(tz=timezone.utc) + timedelta(days=365),
+                datetime(2027, 1, 1, tzinfo=timezone.utc),
+            ),
+        ]
+    )
+
+    application_rounds: list[ApplicationRound] = []
+
+    for i in range(number):
+        criteria = get_paragraphs()
+
+        period = next(period_options)
+
+        application_round = ApplicationRound(
+            name=f"Application Round {i}",
+            name_fi=f"Application Round {i}",
+            name_en=f"Application Round {i}",
+            name_sv=f"Application Round {i}",
+            target_group=random.choice(ApplicationRound.TARGET_GROUP_CHOICES)[0],
+            application_period_begin=period[0],
+            application_period_end=period[1],
+            reservation_period_begin=period[0],
+            reservation_period_end=period[1],
+            public_display_begin=datetime(2021, 1, 1, tzinfo=timezone.utc),
+            public_display_end=datetime(2027, 1, 1, tzinfo=timezone.utc),
+            service_sector=next(service_sectors_loop),
+            criteria=criteria.fi,
+            criteria_fi=criteria.fi,
+            criteria_en=criteria.en,
+            criteria_sv=criteria.sv,
+        )
+        application_rounds.append(application_round)
+
+    application_rounds = ApplicationRound.objects.bulk_create(application_rounds)
+    for application_round in application_rounds:
+        application_round.reservation_units.add(
+            *random_subset(reservation_units, max_size=10)
+        )
+        application_round.purposes.add(*random_subset(reservation_purposes, max_size=5))
+
+    return application_rounds
+
+
+@with_logs(
+    text_entering="Creating applications...",
+    text_exiting="Applications created!",
+)
+def _create_applications(
+    application_rounds: list[ApplicationRound],
+    users: list[User],
+    age_groups: list[AgeGroup],
+    reservation_purposes: list[ReservationPurpose],
+    reservation_units: list[ReservationUnit],
+    *,
+    number: int = 20,
+) -> list[Application]:
+
+    contact_persons = _create_persons(number=number)
+    billing_addresses = _create_addresses(number=number)
+    organisations = _create_organisation(billing_addresses)
+    applications: list[Application] = []
+
+    items: zip[tuple[Person, Address, tuple[str, Organisation | None]]]
+    items = zip(contact_persons, billing_addresses, organisations, strict=True)
+    for contact_person, billing_address, (applicant_type, organisation) in items:
+        # User is the overall admin mode often, but other users are also possible
+        user: User = random.choices(
+            users, weights=[len(users)] + ([1] * (len(users) - 1))
+        )[0]
+
+        application = Application(
+            applicant_type=applicant_type,
+            contact_person=contact_person,
+            user=user,
+            organisation=organisation,
+            application_round=random.choice(application_rounds),
```

Comment:
Statuses for:
- **_ApplicationRound_**: **past** = `HANDLED`, **current** = `DRAFT` [_sic_], **future** = `DRAFT`
- **_Application_**: **past** = `HANDLED` (but decisions not sent) or `CANCELLED` or `EXPIRED`, **current** = `DRAFT` or `RECEIVED` or `CANCELLED`, **future** = _no applications for this kind of round_
- **_ApplicationEvent_**: **past** = `APPROVED` (would be `RESERVED` and the parent statuses `SENT` if process inside the system extended that far) or `DECLINED`, **current** = `CREATED` (i.e., not yet decided/handled)

Reply:
Appropriate status entities are create with the above rules.